packages/types/src/runner.ts

```diff
@@ -0,0 +1,30 @@
+import type { CodeToHastOptions } from 'shiki'
+
+export interface RunnerContext {
```

Comment:
```suggestion
export interface CodeRunnerContext {
```

Let's prefix all `Runner` with `CodeRunner` to be more explict.